Formula/bowtie2.rb

```diff
@@ -15,12 +15,14 @@ class Bowtie2 < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "46f6e5aa64fd0f9564d15f21aa0ac84d9960fb79424539599c284d0f7724e096"
   end
 
-  depends_on "simde"
-  depends_on "tbb"
-
-  uses_from_macos "python"
+  uses_from_macos "perl"
```

Comment:
Does it need a new `perl`? `perl` is pretty much everywhere; it's just hard to guarantee a version...

Reply:
The last time I discussed this, I think the idea was to make sure formulae work on some pared down Linux distros without `perl`. Not sure if this stance has changed.

It would be nice to have an explicit list of system packages (outside of things Homebrew needs to function like `glibc`, `ruby`, ...) that we can assume are installed on Linux. The ones that we seem to assume are GNU tools like `gnu-sed`, `coreutils`, ...

Reply:
I guess it hasn't, but we should consider changing. I'd like to lighten some dependency trees.